addon/-private/closest-string.ts

```diff
@@ -0,0 +1,54 @@
+// https://stackoverflow.com/a/36566052
+function editDistance(s1: string, s2: string) {
```

Comment:
I wonder if we can prune this from prod build.

Reply:
I was able to remove the code, but not the module from the prod build using `@embroider/macros`.

Here's the stats of `npm ci && ember build --environment="production"`
```
9dfb367 (origin/main)
dist/assets/vendor-cc4233d3b7e5776c9b844d7c3d4ad987.js: 453.59 KB (127.62 KB gzipped)

0ead844
dist/assets/vendor-745ebb22057b189d3eb1f47842a4565d.js: 455.02 KB (128.07 KB gzipped)

6592e63
dist/assets/vendor-c13bd7dc8daa2fed4b29e9042c69bd03.js: 454.52 KB (127.82 KB gzipped)
```